DDR4 OC Guide.md

```diff
@@ -579,14 +580,15 @@ The default value is fixed 1.100V and AMD recommends keeping it at that level. I
   * Thanks to Bored and Muren for finding and verifying this on their Asrock motherboards.
 
 ### AMD
-* Try playing around with ProcODT if you can't boot. On Ryzen 1000 and 2000, you should try values between 40Ω and 68.6Ω.  
-On Ryzen 3000, [1usmus](https://www.overclock.net/forum/13-amd-general/1640919-new-dram-calculator-ryzena-1-5-1-overclocking-dram-am4-membench-0-7-dram-bench-480.html#post28049664) suggests 28Ω - 40Ω.  
+* Try playing around with ProcODT if you can't boot. This setting determines the impedence of the On-Die-Termination. According to [Micron](https://www.micron.com/support/~/media/D546161C2C6140BCB0BAEE954AA53433.pdf), higher settings of ProcODT can lead to more stable RAM, but trade off potentially needing higher voltages. On Ryzen 1000 and 2000, you should try values between 40Ω and 68.6Ω due to the considerably weaker memory controller. 
```

Comment:
```suggestion
* Try playing around with ProcODT if you can't boot. This setting determines the processor on-die termination impedance. According to [Micron](https://www.micron.com/support/~/media/D546161C2C6140BCB0BAEE954AA53433.pdf), higher settings of ProcODT can lead to more stable RAM, but trade off potentially needing higher voltages. On Ryzen 1000 and 2000, you should try values between 40Ω and 68.6Ω due to the considerably weaker memory controller. 
```

> you should try values between 40Ω and 68.6Ω due to the considerably weaker memory controller

Source? Sure the IMC is weaker but that isn't necessarily related to needing higher ProcODT. Zen 2 can run ProcODT in that range but it has a stronger IMC.

Reply:
this mainly comes down to the aforementioned micron spec. According to that, higher ODT directly leads to higher signal integrity. Furthermore, a weaker IO die and IMC would have worse signal integrity and would be more susceptible to reflections. I'm not claiming a weaker IMC allows for higher ODT, you're right in saying Zen2 can run at high ODT, but the point is that since the memory subsystem for Zen2 and Zen3 is better, it does not require as high of ODT to compensate.

Reply:
I could rephrase it to memory subsystem instead of IMC if you'd like. btw, considering you trust Yuri, and source him, note that this is something him and I have discussed before.